docs/src/extend/plugins.md

```diff
@@ -36,6 +36,8 @@ export default plugin;
 module.exports = plugin;
 ```
 
+If you plan to distribute your plugin as an npm package, make sure that the module that exports the plugin object is the default export of your package. This will enable ESLint to import the plugin when it is specified in the command line in the `--plugin` option.
```

Comment:
[Docs] Maybe wrap <code>`--plugin` option</code> in a link to https://eslint.org/docs/latest/use/command-line-interface#--plugin?

Reply:
Thanks for the suggestions, it's all done in b138d4682abaeb313e74c4ef22ffc3477a7310cf.